frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -105,10 +72,41 @@ class PopupCreateFile extends React.Component {
     }
   })
 
+  handleUploadedEnd = () => {
+    const { state, props } = this
+
+    let uploadedFileFailedList = state.uploadedFiles
```

Comment:
merge this line and the line bellow into a single one. That's what immutable functions are for.
You can then change `uploadedFileFailedList` to be a const